up42/initialization.py

```diff
@@ -33,12 +34,20 @@
 logger = get_logger(__name__, level=logging.INFO)
 
 
+def _check_deprecated_implicit_project_id(project_id: Optional[str]):
+    if not project_id:
+        warn("Implicit project_id is deprecated", DeprecationWarning, stacklevel=2)
```

Comment:
We should enhance this warning.
From a user's perspective, what does it mean? What do I need to do when I see this warning?

Reply:
@seedlit agree, I assumed this will be handled by the documentation team

Reply:
@up42/documentation

Reply:
[Updated](https://github.com/up42/up42-py/compare/master...AT-3556_AuthMigration#diff-1dd53e0a04ae90a179c0b71fcb33ed9137a11fbe3ce591ea1f1701999bf38a69R39)